adam-hbase/src/test/resources/small.vcf

```diff
@@ -0,0 +1,57 @@
+##fileformat=VCFv4.1
+##FILTER=<ID=IndelFS,Description="FS > 200.0">
```

Comment:
You don't need to copy this file, you should be able to use the `copyResource` function in `ADAMFunSuite`.